environment/swarmball_env.py

```diff
@@ -4,17 +4,24 @@
 from .simulation.simulation import SwarmBallSimulation
 
 class SwarmBall(gym.Env):
-    def __init__(self):
-        self.sim = SwarmBallSimulation()
+    def __init__(self, acc_factor=0.25, thresh_amount=3, **kwargs):
+        self.sim = SwarmBallSimulation(**kwargs)
+        self.thresholds = [[] for _ in range(thresh_amount)]
+        self.thresh_vel = [0,0,0]
+        self.acc_factor = acc_factor
 
-    def seed(self, seed=None):
-        """
-            To przerażające nasiono!!!
-        """
+    def reward(self):
         pass
 
     def step(self, action):
+        self.thresholds.pop(0)
+        self.thresholds.append(self.sim.threshold_positions())
+        self.thresh_vel = [self.thresh_vel[i] + action[i]*self.acc_factor for i in range(len(self.thresh_vel))]
+        for i in range(len(action)):
+            self.sim.update_thresholds_position(i, self.sim.threshold_positions()[i] + self.thresh_vel[i])
```

Comment:
Czy update po sczytaniu (mam nadzieję, że tak to się pisze) jest tym co chcemy zrobić? W ten sposób chyba zwrócimy w obserwacji położenie thresholdów przed wykonaniem akcji?

Reply:
Racja. Pisane było kiedy jeszcze myślałem że podajecie nowe położenie. Wtedy bez sensu wam zwracać to co podaliście. Ale dla przyśpieszenia zmienię.